contracts/test/mocks/SafeERC20Mock.sol

```diff
@@ -0,0 +1,28 @@
+pragma solidity 0.4.24;
+
+import "../../common/SafeERC20.sol";
+import "../../lib/token/ERC20.sol";
+
+
+contract SafeERC20Mock {
+    using SafeERC20 for ERC20;
+    event Result(bool result);
+
+	function transfer(ERC20 token, address to, uint256 amount) external returns (bool) {
```

Comment:
You know I don't mind these things ;-), but just in case you didn't notice there seems to be a mix of 4 and 2 spaces indentation here.